src/System.ServiceModel.UnixDomainSocket/src/System/ServiceModel/UnixDomainSocketSecurity.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.Diagnostics.Contracts;
+using System.Runtime.InteropServices;
+using System.ServiceModel.Channels;
+
+namespace System.ServiceModel
+{
+    public sealed class UnixDomainSocketSecurity
+    {
+        internal const UnixDomainSocketSecurityMode DefaultMode = UnixDomainSocketSecurityMode.Transport;
+
+        private UnixDomainSocketSecurityMode _mode;
+
+        public UnixDomainSocketSecurity()
+        {
```

Comment:
Instead of doing this inline, make DefaultMode a private static field and use the original constructor. Then you can use the default mode in the exception message of the private constructor